tests/api_tests/nodes/test_views.py

```diff
@@ -1349,3 +1351,115 @@ def test_return_deleted_private_node_pointer(self):
+        assert_less(dt1 - dt2, dt.timedelta(milliseconds=allowance))
+
+    def test_log_create_on_public_project(self):
+        datetime.datetime.strptime('2015-07-28 21:06:34.965114','%Y-%m-%d %H:%M:%S.%f')
+        date_str = "2008-11-10 17:53:59"
+        dt_obj = datetime.datetime.strptime(date_str, "%Y-%m-%d %H:%M:%S")
+        print repr(dt_obj)
+        res = self.app.get(self.public_url)
+        assert_equal(res.status_code, 200)
+        (len(res.json['data']), 1)
+        assert_datetime_equal(datetime.datetime.strptime(res.json['data'][0]['date'], "%Y-%m-%dT%H:%M:%S.%f"),
+                              self.public_project.logs[0].date)
+        assert_equal(res.json['data'][0]['id'], self.public_project.logs[0]._id)
+        assert_equal(res.json['data'][0]['action'], self.public_project.logs[0].action)
+        assert_equal(res.json['data'][0]['version'], self.public_project._version)
+        assert_equal(res.json['data'][0]['name'], self.public_project.logs[0]._name)
+
+    def test_log_create_on_private_project(self):
+        res = self.app.get(self.private_url, auth=self.basic_auth)
+        assert_equal(res.status_code, 200)
+        (len(res.json['data']), 1)
+        assert_datetime_equal(datetime.datetime.strptime(res.json['data'][0]['date'], "%Y-%m-%dT%H:%M:%S.%f"),
+                              self.private_project.logs[0].date)
+        assert_equal(res.json['data'][0]['id'], self.private_project.logs[0]._id)
+        assert_equal(res.json['data'][0]['action'], self.private_project.logs[0].action)
+        assert_equal(res.json['data'][0]['version'], self.private_project.logs[0]._version)
+        assert_equal(res.json['data'][0]['name'], self.private_project.logs[0]._name)
+
+    def test_return_logs_private_project_details(self):
+        res = self.app.get(self.private_url, auth=self.basic_auth_two, expect_errors=True)
+        assert_equal(res.status_code, 403)
+
+    def test_log_add_contributor_on_public_project(self):
+        res = self.app.get(self.public_url)
+        assert_equal(res.status_code, 200)
+        (len(res.json['data']), 1)
+        # 1 project create log, 1 add contributor log, then 5 generated logs
+        assert_equal(res.json['links']['first'], None)
+        assert_equal(res.json['links']['last'], None)
+        assert_equal(res.json['links']['prev'], None)
+        assert_equal(res.json['links']['next'], None)
+        assert_equal(res.json['links']['meta']['total'], 1)
+        assert_equal(res.json['links']['meta']['per_page'], 10)
+
+    def test_get_logs_defaults_to_ten(self):
+        res = self.app.get(self.public_url)
+        (len(res.json['data']), 5)
+        # 1 project create log, 1 add contributor log, then 5 generated logs
+        assert_equal(res.json['links']['meta']['total'], 1)
+        assert_equal(res.json['links']['meta']['total'], 1)
+        assert_equal(res.json['links']['meta']['per_page'], 10)
+
+    def test_get_more_logs(self):
```

Comment:
We don't need this test. Also, it doesn't look correct, because if there were 12 items and only 10 per page, there should be 2 total pages.